src/components/atoms/Plugin/PluginIFrame/useIFrame.ts

```diff
@@ -69,6 +71,10 @@ export default function useIFrame({
     if (!ready) setIFrameLoaded(false);
   }, [ready]);
 
+  useEffect(() => {
+    if (!enabled) setIFrameLoaded(false);
+  }, [enabled]);
```

Comment:
Can you combine this with the useEffect already there on line 70? Then you can just do
```
 useEffect(() => {
    if (!ready || !enabled) setIFrameLoaded(false);
  }, [ready, enabled]);
```

Reply:
Updated.